src/components/ui/ListEntry.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
```

Comment:
Great work. Definitely looks better after Friday, but please check with Siraj to see if there are any ways you can further improve this CSS.